src/frontends/common/include/common/telemetry_extension.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <type_traits>
+
+#include "frontend_defs.hpp"
+#include "openvino/core/extension.hpp"
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/pass.hpp"
+
+namespace ov {
+namespace frontend {
+
+/// \brief Provides callback to report telemetry information back to Python code
+class FRONTEND_API TelemetryExtension : public ov::Extension {
+public:
+    using error_callback = std::function<void(const std::string& category, const std::string& error_message)>;
+    using event_callback = std::function<
+        void(const std::string& category, const std::string& action, const std::string& label, int value)>;
+    TelemetryExtension(const std::string& event_category,
+                       const event_callback& send_event,
```

Comment:
The live time of `event_callbacks` should be longer than `TelementryExtension`

Reply:
It will be fixed in the next PR, 
ticket 72263